tsl/test/expected/dist_hypertable-11.out

```diff
@@ -13,9 +13,9 @@ SELECT (add_data_node (name, host => 'localhost', DATABASE => name)).*
 FROM (VALUES (:'DATA_NODE_1'), (:'DATA_NODE_2'), (:'DATA_NODE_3')) v (name);
      node_name     |   host    | port  |     database      | node_created | database_created | extension_created 
 -------------------+-----------+-------+-------------------+--------------+------------------+-------------------
- dist_hypertable_1 | localhost | 55432 | dist_hypertable_1 | t            | t                | t
- dist_hypertable_2 | localhost | 55432 | dist_hypertable_2 | t            | t                | t
- dist_hypertable_3 | localhost | 55432 | dist_hypertable_3 | t            | t                | t
+ dist_hypertable_1 | localhost | 15432 | dist_hypertable_1 | t            | t                | t
```

Comment:
This change seems unrelated to the PR. I think the old port number is cached in your build directory and you need to recreate it.